docs/api.md

```diff
@@ -4552,4 +4816,50 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Property Detailed (object)
 + name: `first_name` (string, required) - The name of the Property.
 + publishedAt: `2018-01-21T00:04:11.153Z` (string, required) - Publishing timestamp of the form that defined this property for the first time.
-+ forms: (array[Form KeyValue]) - List of forms that create the property
\ No newline at end of file
++ forms: (array[Form KeyValue]) - List of forms that create the property
+
+## EntitySummaryFields (object)
++ uuid: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `uuid` of the Entity that uniquely identifies the Entity.
++ datasetName: `people` (string, required) - Name of the dataset
++ createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format. The time that the server received the Entity.
++ updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the last update in ISO date format. `null` when there is only one version of the Entity.
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the deletion in ISO date format. `null` if the Entity is not deleted.
+
+## EntitySourceType (enum)
++ SUBMISSION (string) - Entity was created through a Form Submission.
++ API (string) - Entity was created using the API.
++ FILE_UPLOAD (string) - Entity was created through uploading a file.
+
+## EntitySource (object)
++ type: `API` (EntitySourceType, required) - From what type of source this Entity was created.
++ id: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - `FormXmlId`/`SubmissionID` if the type is Submission otherwise value of `X-Client-Id` provided at the time of Entity creation.
```

Comment:
Oh and it looks like the release criteria call for the submission instance name to be shown. Is it possible for the source here to include that as well?

Reply:
I have added `details` object but frontend would call `/diffs` API to generate entity feed.

Also `projectId` is not necessary because API consumer already knows it - it is required in URL

Reply:
I'm thinking that Frontend would use `entity.currentVersion.source.details` in the "Basic Information" section of the page to show "Creating Submission". Could we add a `deletedAt` timestamp to the `details` object as well in case the form is soft-deleted or purged?

If the form is purged, will we be able to return the submission's instance name? I don't think we save that anywhere now, and I'm not sure we'd want to. Sometimes data is purged because it's sensitive. Maybe Frontend should just show the instance ID in that case? 🤔

Reply:
> Could we add a `deletedAt` timestamp to the `details` object as well in case the form is soft-deleted or purged?

There's `forms.deletedAt` and `actees.purgedAt`. Maybe we return them as two separate timestamps?

Reply:
I meant to say `/entities/:uuid/audits` here

> I have added details object but frontend would call /diffs API to generate entity feed.

We can return `forms.deletedAt` and `actees.purgedAt` with the `/audits`. 

Related to this, which `instanceName` is required on the frontend. Instance name at the time of entity creation or current instance name?

Reply:
> We can return `forms.deletedAt` and `actees.purgedAt` with the `/audits`.

Sounds good to me. :+1:

> Related to this, which `instanceName` is required on the frontend. Instance name at the time of entity creation or current instance name?

I don't think the release criteria say, but my instinct is that it should be the current instance name. If the user clicks the link to the submission, that page is going to show the current instance name. If that's hard, I also think we could consider having Frontend send an extra/sequential request for the submission and its current instance name (and just show a loading message in the meantime for "Creating Submission").